fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkWriter.java

```diff
@@ -401,11 +442,122 @@ public OperationOutcome conditionalFingerprintUpdate(ImportTransientUserData chu
             long endTime = System.currentTimeMillis();
             String location = "@source:" + ctx.getSource() + "/" + ctx.getImportPartitionWorkitem();
             if (!skipped) {
-                auditLogger.logUpdateOnImport(resource, new Date(startTime), new Date(endTime), status, location, "BulkDataOperator");
+                auditLogger.logUpdateOnImport(resource, new Date(startTime), new Date(endTime), status, location,
+                        "BulkDataOperator");
             } else {
-                auditLogger.logUpdateOnImportSkipped(oldResource, new Date(startTime), new Date(endTime), status, location, "BulkDataOperator");
+                auditLogger.logUpdateOnImportSkipped(oldResource, new Date(startTime), new Date(endTime), status,
+                        location, "BulkDataOperator");
             }
         }
         return oo;
     }
+
+    /**
+     * Generates ALL_OK along with a line number.
+     * 
+     * @param lineNumber the location in the file.
+     * @return
+     */
+    private OperationOutcome generateAllOkValidation(long lineNumber) {
+        return OperationOutcome.builder()
+                .issue(Issue.builder()
+                        .severity(IssueSeverity.INFORMATION)
+                        .code(IssueType.INFORMATIONAL)
+                        .details(
+                                CodeableConcept.builder()
+                                        .text(string("All OK"))
+                                        .build())
+                        .extension(Extension.builder()
```

Comment:
did you consider using Issue.diagnostics?
I guess the advantage of the extension is its always gonna be a number this way, so less "standard" but more computable...

Reply:
Yeah - I intended it to be specifically interpreted.